qemu/target-arm/helper.c

```diff
@@ -155,7 +155,7 @@ static void count_cpreg(gpointer key, gpointer opaque)
     }
 }
 
-static gint cpreg_key_compare(gconstpointer a, gconstpointer b)
+static int32_t cpreg_key_compare(gconstpointer a, gconstpointer b)
```

Comment:
i think `gint` is `int`, but not int32_t ? see https://developer.gnome.org/glib/stable/glib-Basic-Types.html#gint